opal/runtime/opal_params.c

```diff
@@ -153,6 +155,50 @@ int opal_register_params(void)
 	return ret;
     }
 
+    opal_abort_delay = 0;
+    ret = mca_base_var_register("opal", "opal", NULL, "abort_delay",
+                                "If nonzero, print out an identifying message when abort operation is invoked (hostname, PID of the process that called abort) and delay for that many seconds before exiting (a negative delay value means to never abort).  This allows attaching of a debugger before quitting the job.",
+                                 MCA_BASE_VAR_TYPE_INT, NULL, 0, 0,
+                                 OPAL_INFO_LVL_9,
+                                 MCA_BASE_VAR_SCOPE_READONLY,
+                                 &opal_abort_delay);
+    if (0 > ret) {
+	return ret;
+    }
+    (void) mca_base_var_register_synonym(ret, "ompi", "mpi", NULL,
+                                         "abort_delay",
+                                         MCA_BASE_VAR_SYN_FLAG_DEPRECATED);
+    (void) mca_base_var_register_synonym(ret, "oshmem", "oshmem", NULL,
+                                         "abort_delay",
+                                         MCA_BASE_VAR_SYN_FLAG_DEPRECATED);
+
+    opal_abort_print_stack = false;
+    ret = mca_base_var_register("opal", "opal", NULL, "abort_print_stack",
+                                 "If nonzero, print out a stack trace when abort is invoked",
+                                 MCA_BASE_VAR_TYPE_BOOL, NULL, 0,
+                                /* If we do not have stack trace
+                                   capability, make this a constant
+                                   MCA variable */
+#if OPAL_WANT_PRETTY_PRINT_STACKTRACE
+                                 0,
+                                 OPAL_INFO_LVL_9,
+                                 MCA_BASE_VAR_SCOPE_READONLY,
+#else
+                                 MCA_BASE_VAR_FLAG_DEFAULT_ONLY,
+                                 OPAL_INFO_LVL_9,
+                                 MCA_BASE_VAR_SCOPE_CONSTANT,
+#endif
+                                 &opal_abort_print_stack);
+    if (0 > ret) {
+	return ret;
+    }
+    (void) mca_base_var_register_synonym(ret, "ompi", "mpi", NULL,
+                                         "abort_print_stack",
+                                         MCA_BASE_VAR_SYN_FLAG_DEPRECATED);
+    (void) mca_base_var_register_synonym(ret, "oshmem", "oshmem", NULL,
```

Comment:
You can't register these synonyms down here in OPAL -- OPAL doesn't know anything about OMPI or OSHMEM.  Please move these alias registrations up to their respective layers.